technology/stakeholder-approved-funding.php

```diff
@@ -19,7 +19,7 @@
 
     <!--  Open Graph Tags -->
     <meta property="og:title" content="BTS Holder Approved Funding | BitShares Blockchain">
-    <meta property="og:description" content="BitShares blockchain is self-funding and has a reserve pool self-sustained by fees, whereby BTS Holder Approved Funding direct where blockchain budgets go.">
+    <meta property="og:description" content="BitShares blockchain is self-funding and has a reserve pool sustained by fees, whereby BTS Holder Approved Funding direct where blockchain budgets go.">
```

Comment:
Improve description. It needs to describe page or be more user-friendly, easy to understand.